src/Psalm/Internal/Analyzer/Statements/Expression/AssertionFinder.php

```diff
@@ -3067,6 +3129,7 @@ private static function getFalseEqualityAssertions(
             }
         }
 
+        /** @psalm-suppress LessSpecificReturnStatement */
```

Comment:
Could you add the shape inferred by Psalm at the end of those suppressions please?

Reply:
yes please, there may be a good reason to suppress that but it's basically impossible to review once the suppression has been made